src/content/docs/faqs/common-questions-for-developers.md

```diff
@@ -1,6 +1,74 @@
 ---
-title:  "Common Questions for Developers"
-description:  "Common Questions for Developers"
+title: "Developers FAQ"
+description: "Frequently Asked Questions tailored for developers working with Angor."
 sidebar:
-  order: 3
+  order: 4
 ---
+
+## **What is Angor, and why is it relevant for developers?**
+Angor is a decentralized protocol built on Bitcoin and Nostr. It offers developers the opportunity to contribute to an open-source project that empowers secure crowdfunding without intermediaries.
+
+![bitcoin](./bitcoin2.svg)
+
+## **How can I contribute to Angor’s development?**
+You can:
+1. Fork the open-source repository from GitHub.
+2. Submit pull requests for new features or improvements.
+3. Join discussions and contribute to the roadmap on community platforms.
+
+![bitcoin](./bitcoin2.svg)
+
+## **What technologies are used in Angor?**
+Angor leverages:
+- **Bitcoin scripting:** For smart contracts and fund security.
+- **Nostr protocols:** For metadata storage and communication.
+- **Open-source tools:** To ensure transparency and scalability.
+
+![bitcoin](./bitcoin2.svg)
+
+## **How do I set up a local development environment for Angor?**
+1. Clone the Angor repository.
+2. Install dependencies (e.g., Bitcoin Core, Nostr relays).
+3. Follow the setup guide in the documentation.
+
+![bitcoin](./bitcoin2.svg)
+
+## **What is the role of Nostr in Angor?**
+Nostr is used for:
+- Storing project metadata.
+- Enabling secure communication between founders and investors.
+- Supporting the decentralized nature of Angor.
+
+![bitcoin](./bitcoin2.svg)
+
+## **How can I integrate custom features into Angor?**
+1. Identify the feature’s purpose and design a solution.
+2. Submit a proposal to the Angor development team.
+3. Implement the feature following the coding standards and guidelines.
+
+![bitcoin](./bitcoin2.svg)
+
+## **Does Angor have a plugin system for developers?**
+Yes. Developers can create custom plugins for extended functionality, such as additional communication tools or analytics.
```

Comment:
Do we have a plugin in system? I don't think this is true.
Perhaps what devs can do is build anew nostr client for angor functionality but this is not  plugins